### ISSUE TITLE
Taunt problem

### ISSUE THREAD

**Issue by windroidx** [naydef/Afterlife-plugin#3]
player in neutral team can use taunt to affect the alive players.
Some player can use Huntsman taunt to alive players that make them stunned.

Maybe the plugin should blocking all taunts or just weapon taunts for neutral team?

**Comment by naydef** [comment]
Yep. I know for the problem, but I left it because it's funny to RPS with the real team. If this is significant problem for you, I can take care about it!

Edit: You mean taunt with some weapons. Will fix it (very) soon
